bokeh/model.py

```diff
@@ -321,6 +321,13 @@ def id(self) -> str:
 
     """)
 
+    syncable: bool = Bool(default=True, help="""
+    Indicates whether this model should be synchronized when updated in a web browser. This
+    is useful to set when dealing with frequently updated objects, which updated values we
+    don't need, reducing unnecessary network traffic. Note setting this will result in an
+    inconsistent state of objects and any callbacks set on this object will not trigger.
```

Comment:
```suggestion
    Whether this model should be synchronized back to a Bokeh server when updated in a 
    web browser. Setting to False may be useful to reduce network traffic when dealing with 
    frequently updated objects whose updated values we don't need.
    
    .. note:: 
        Setting this property to False will prevent any ``on_change`` callbacks on this object 
        from triggering.
```

Reply:
Also we should be explicit: JS callbacks will still tigger, or no?

Reply:
This is only about synchronization, so JS callbacks should trigger. I will make this clearer.